agent/src/main/java/com/code_intelligence/jazzer/api/MethodHook.java

```diff
@@ -23,11 +23,12 @@
 import java.lang.invoke.MethodType;
```

Comment:
Could you also update the javadocs at https://github.com/CodeIntelligenceTesting/jazzer-api? I submitted a PR to get the instructions updated.

Reply:
Ah, was not aware the we have to do that. I will create a PR for the update.